src/v2/es-utils.js

```diff
@@ -95,6 +101,21 @@ class ESClass {
     return response;
   }
 
+  async delDocById(index, type, id) {
+    let exists = false;
+    if (id) {
+      // Check if it already exists
+      exists = await this.client
+        .exists({ index: index, type: type, id: id });
+    }
+    if (exists) {
+      const response = await this.client
+        .delete({ index: index, type: type, id: id });
+      return response;
+    }
+    return { _id: id, result: 'not_exists', statusCode: 200, reason: 'Does not exist' };
```

Comment:
Should this be a 404?

Reply:
This one is not an error. It just informs the user that an item didn't exist in Kibana and therefore wasn't deleted